applications/asset_tracker_v2/src/cloud/nrf_cloud_integration.c

```diff
@@ -255,10 +258,10 @@ int cloud_wrap_init(cloud_wrap_evt_handler_t event_handler)
 	}
 
 	/* Set null character at the end of the device IMEI. */
-	imei_buf[NRF_CLOUD_CLIENT_ID_LEN] = 0;
-
-	strncpy(client_id_buf, imei_buf, sizeof(client_id_buf) - 1);
+	imei_buf[IMEI_LEN] = 0;
 
+	strncat(client_id_buf, imei_buf,
+		(sizeof(client_id_buf) - CLOUD_CLIENT_ID_IMEI_PREFIX_LEN - 1));
```

Comment:
Am I missing something, or could this be simplified to `IMEI_LEN`?

~~(sizeof(client_id_buf)~~ - CLOUD_CLIENT_ID_IMEI_PREFIX_LEN - 1
= **~~NRF_CLOUD_CLIENT_ID_LEN~~ + 1** - CLOUD_CLIENT_ID_IMEI_PREFIX_LEN - 1
= **IMEI_LEN +** ~~**CLOUD_CLIENT_ID_IMEI_PREFIX_LEN** + 1 - CLOUD_CLIENT_ID_IMEI_PREFIX_LEN - 1~~
= IMEI_LEN

Reply:
Nice catch! Thanks.